src/Http/Http.Extensions/gen/DiagnosticDescriptors.cs

```diff
@@ -24,4 +24,16 @@ internal static class DiagnosticDescriptors
         DiagnosticSeverity.Warning,
         isEnabledByDefault: true
     );
+
+    // This is temporary. The plan is to be able to resolve all parameters to a known EndpointParameterSource.
+    public static DiagnosticDescriptor GetUnableToResolveParameterDescriptor(string parameterName)
+    {
+        return new(
+            "RDG073",
```

Comment:
```suggestion
            "RDG003",
```

Reply:
I didn't want to burn a real diagnostic id since I plan this to be temporary.